build/webpack.base.config.js

```diff
@@ -13,33 +13,47 @@ const isProd = process.env.NODE_ENV === 'production';
 const isDev = !isLocal && !isProd;
 const logLevel = process.env.LOG_LEVEL || 'debug';
 const environment = isProd ? 'production' : 'development';
+const autoprefixerEnabled = process.env.CSS_AUTOPREFIXER_ENABLED === 'true';
```

Comment:
If this environment variable is set to true, we include the `postcss-loader` and update the importLoaders count accordingly.